### DISCUSSION TITLE
Installed using go

### DISCUSSION THREAD

**Discussion by mikaelweiss** [jesseduffield/lazygit#1192] [Q&A] [answered]
I installed using go and it didn't seem to work. I type ``lazygit`` enter and then I get the error ``zsh: command not found: lazygit``
Any ideas about what might be up?

**Comment by Mic92** [comment, marked as answer]
You need to add `$GOPATH/bin` to your `$PATH`

**Reply by mikaelweiss** [reply]
Awesome. That fixed it. Thank you!
